pkg/kapis/devops/v1alpha3/handler.go

```diff
@@ -327,3 +327,14 @@ func (h *devopsHandler) getDevOps(request *restful.Request) (operator devops.Dev
 	}
 	return
 }
+
+func (h *devopsHandler) BuildPipelineParameters(request *restful.Request, response *restful.Response) {
+	devopsQuery := request.PathParameter("devops")
+	pipelineQuery := request.PathParameter("pipeline")
+	if client, err := h.getDevOps(request); err == nil {
+		parameters, err := client.BuildPipelineParameters(devopsQuery, pipelineQuery, request.Request.URL.Query())
```

Comment:
I'm thinking it would be better if return an empty array when not any parameters. Please see my test results:

![image](https://user-images.githubusercontent.com/1450685/163741651-6615ec86-de7f-41dc-91bd-e29da29424e9.png)

`null` as a response is not good enough for the front-end.